active_learning/liver_tumor_al/results_uncertainty_analysis.ipynb

```diff
@@ -0,0 +1,421 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ff8040b5",
+   "metadata": {},
+   "source": [
+    "Copyright (c) MONAI Consortium  \n",
+    "Licensed under the Apache License, Version 2.0 (the \"License\");  \n",
+    "you may not use this file except in compliance with the License.  \n",
+    "You may obtain a copy of the License at  \n",
+    "&nbsp;&nbsp;&nbsp;&nbsp;http://www.apache.org/licenses/LICENSE-2.0  \n",
+    "Unless required by applicable law or agreed to in writing, software  \n",
+    "distributed under the License is distributed on an \"AS IS\" BASIS,  \n",
+    "WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  \n",
+    "See the License for the specific language governing permissions and  \n",
+    "limitations under the License.\n",
+    "\n",
+    "# MONAI Active Learning Uncertainty Exploration & Results Analysis\n",
+    "\n",
+    "There are two sections in this notebook:\n",
+    "    \n",
+    "1.) How to visualize uncertainty from given unlabeled samples\n",
+    "\n",
+    "2.) How to analyze results once the active learning scripts have been run to completion"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "36031dd0",
+   "metadata": {},
+   "source": [
+    "## Setup environment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "9c089995",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python -c \"import monai\" || pip install -q \"monai-weekly[pillow, tqdm]\"\n",
+    "!python -c \"import matplotlib\" || pip install -q matplotlib\n",
+    "!python -c \"import pickle\" || pip install -q pickle\n",
```

Comment:
There is no need to `pip install pickle`, as it is included by python >= 3.7

Reply:
removed.